runtime/error.c

```diff
@@ -235,6 +240,18 @@ static void printBacktraceLine(const char *filename, int64_t lineNumber, const c
     fflush(fp);
 }
 
+static bool isPrefix(const char* prefix, const char* str) {
```

Comment:
You can write this so that it looks at each character of prefix and str at most once. (Don't have calls to strlen.)

Reply:
fixed